app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
hmmm, that's a good point but I think we have the flexibility to do either approach if it is in `/lib/services` and for now maybe just do implicit requires? that is an easy performance tweak should it become a problem I would think.